plugins/Contents/tests/System/expected/test_Contents__Actions.getPageUrls_month.xml

```diff
@@ -26,6 +26,6 @@
 		<exit_rate>100%</exit_rate>
 		<avg_time_generation>0.333</avg_time_generation>
 		<url>http://www.example.org/page</url>
-		<segment>pageUrl==http%3A%2F%2Fwww.example.org%2Fpage</segment>
+		<segment>pageUrl==http%253A%252F%252Fwww.example.org%252Fpage</segment>
```

Comment:
is this double encoded on purpose?

Reply:
yes, because one decode is for the condition, and one for the value

Reply:
I don't think it's correct, would expect to have here either:
`pageUrl==http%253A%252F%252Fwww.example.org%252Fpage` (current behavior)
or
`pageUrl%3D%3Dhttp%253A%252F%252Fwww.example.org%252Fpage` (current behavior URL encoded)

but `pageUrl==http%253A%252F%252Fwww.example.org%252Fpage` wouldn't be a correct value since Matomo would then segment based on Page URL `http%3A%2F%2Fwww.example.org%2Fpage` which wouldn't exist (didn't check but it's possible it currently works anyway since we try the url decoded version in https://github.com/matomo-org/matomo/blob/2020b122615789c36eaa0917e10b483f12793b3c/core/Segment.php#L99-L108 but still: we wouldn't want to show this double encoded URL in the API output)

Reply:
It is required to have the value encoded 3 times because it is urldecoded 3 times, see the PR description for more info.